src/McBopomofo.cpp

```diff
@@ -411,6 +455,8 @@ void McBopomofoEngine::handleCandidatesState(
       std::make_unique<fcitx::CommonCandidateList>();
 
   auto keysConfig = config_.selectionKeys.value();
+  selectionKeys_.clear();
```

Comment:
I just realized I removed this line by accident in https://github.com/openvanilla/fcitx5-mcbopomofo/commit/dc00fdcb9bd4ee6a1c43a9e6320d254c5be3468c 😱

Merging this PR now.

Reply:
Awesome! This fixes the very long candidate list issue!